### DISCUSSION TITLE
Unable to build docker container

### DISCUSSION THREAD

**Discussion by 3vts** [saleor/saleor#8014] [Deployment]
I'm trying to build the application with the steps described on the installation guide I already have Docker 
After i run this steps:
`git clone https://github.com/mirumee/saleor-platform.git --recursive --jobs 3`
`cd saleor-platform`
`docker-compose build`

I get this error
`saleor-platform/local.yml: no such file or directory`
Ideas?

**Comment by karolkielecki** [comment]
@3vts Hi, thanks for letting us know. We are aware that users experience some issues. We'll update `saleor-platform` in the upcoming weeks. Once it's updated, I'll let you know.

Cheers,

**Reply by 3vts** [reply]
Is there a way to get it up and running with Docker right now?
